http_logs/index.json

```diff
@@ -1,7 +1,8 @@
 {
   "settings": {
     "index.number_of_shards": {{number_of_shards | default(5)}},
-    "index.number_of_replicas": {{number_of_replicas | default(0)}}
+    "index.number_of_replicas": {{number_of_replicas | default(0)}},
+    "index.requests.cache.enable": false
```

Comment:
We need to define this only in the index definition because the index settings defined on the `create-index` operation are actually *merged* with the index definition in `index.json`. So, if you run the `append-sorted-no-conflicts` in `geonames` you'll get the following index settings:

```json
    "settings" : {
      "index" : {
        "number_of_shards" : "5",
        "provided_name" : "geonames",
        "creation_date" : "1560341111741",
        "sort" : {
          "field" : [
            "country_code.raw",
            "admin1_code.raw"
          ],
          "order" : [
            "asc",
            "asc"
          ]
        },
        "store" : {
          "type" : "hybridfs"
        },
        "number_of_replicas" : "0",
        "uuid" : "1CIlIryVQT2ti0oGjhv6xA",
        "version" : {
          "created" : "8000099"
        }
      }
    }
```

`number_of_shards`, `store.type` and `number_of_replicas` are provided by `index.json` and `index.sort.field` as well as `index.sort.order` are provided by the operation `create-index`.

This means that `index.requests.cache.enable` is in fact explicitly specified in all cases.

Reply:
oh right, I keep thinking it gets overwritten and not merged for some reason. Thank you!